views/doc.list.html.twig

```diff
@@ -3,36 +3,50 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <section role="main">
+
+        <div id="searchbar" class="clearfix">
+            <input id="docsearch" type="text" placeholder="Type here to search documentation..." role="searchbox"
+                   title="Enter text here and submit to search the documentation"/>
+        </div>
+
+        <h1 role="sectionhead">📚 Book</h1>
+        <ul>
+            {% for filename, data in book if book is defined %}
```

Comment:
this is wrong. Checking whether `book is defined` inside the loop over `book` does not make any sense:

- if the variable is not defined, things will blow up when starting the loop
- if the variable is defined, this will stay true for all items of the loop.

Btw, this Twig syntax is deprecated now.

Reply:
SNAP! Oh you're so right, don't even remember why I wrote this part. Will remove! :)

Reply:
Closed: changed the HTML markup accordingly.